common/shell-local/config.go

```diff
@@ -19,14 +20,17 @@ import (
 type Config struct {
 	common.PackerConfig `mapstructure:",squash"`
 
+	shell.Provisioner `mapstructure:",squash"`
+
 	// ** DEPRECATED: USE INLINE INSTEAD **
 	// ** Only Present for backwards compatibility **
 	// Command is the command to execute
 	Command string
```

Comment:
[This should probably be another PR?] Should we get rid of Command here with a fixer; since it is deprecated ? 🤔

Reply:
Yeah, we could probably ditch it now; it's been over a year since I made the change.